code/game/objects/items/plushes.dm

```diff
@@ -595,6 +595,21 @@
 	squeak_override = list('sound/effects/blobattack.ogg' = 1)
 	gender = FEMALE	//given all the jokes and drawings, I'm not sure the xenobiologists would make a slimeboy
 
+/obj/item/toy/plush/slimeplushie/annie
+	name = "slime plushie"
+	desc = "An adorable stuffed toy that resembles a slimey crewmember."
+	icon_state = "annie"
+	item_state = "annie"
+	attack_verb = list("blorbled", "slimed", "absorbed", "glomped")
```

Comment:
Honestly, you should move the "glomped" attack string to the parent type and remove this line altogether.